src/zeit/content/volume/interfaces.py

```diff
@@ -37,47 +44,87 @@ class IVolume(zeit.cms.content.interfaces.IXMLContent):
         max=53)
 
     teaserText = zope.schema.Text(
-        title=_("Teaser text"),
+        title=_("Volume text"),
         required=False,
         max_length=170)
 
     date_digital_published = zope.schema.Datetime(
         title=_('Date of digital publication'),
         required=False)
 
-    covers = zope.interface.Attribute(
-        'Convenience method to adapt to IVolumeCovers')
+    # covers = zope.interface.Attribute(
+    #     'Convenience method to adapt to IVolumeCovers')
 
     previous = zope.interface.Attribute(
         'The previous IVolume object (by date_digital_published) or None')
 
     next = zope.interface.Attribute(
         'The next IVolume object (by date_digital_published) or None')
 
+    def get_cover(name, product_id=None):
+        pass
+
+    def set_cover(name, product_id, value):
+        pass
+
     def fill_template(text):
         """Fill in a string template with the placeholders year=self.year
         and name=self.volume (zero-padded to two digits), e.g.
         ``http://xml.zeit.de/{year}/{name}/ausgabe``.
         """
 
+    # Maybe its nicer use the product (z.cms.contentsources:Product),
+    # instead of product id?
+    def get_cover(cover_id, product_id, use_fallback):
+        """
+        Get a cover of a product.
+        For example volume.get_cover('printcover','ZEI') returns the
+        printcover of DIE ZEIT of this specific volume.
+        If no product_id is given or if no Cover is found and use_fallback
+        is set, the method looks for a cover of the main_product.
+        :param cover_id: str cover ID set in volume-covers.xml
+        :param product_id: str product ID set in products.xml
+        :param product_id: bool specifies if a fallback should be used.
+        :return: zeit.content.image.interfaces.IImageGroup or None
+        """
 
-class VolumeSource(zeit.cms.content.contentsource.CMSContentSource):
+    def set_cover(cover_id, product_id, image):
+        """
+        Set an image as a cover of product.
+        """
 
-    check_interfaces = (IVolume,)
-    name = 'volume'
 
-VOLUME_SOURCE = VolumeSource()
+class ICovers(zope.interface.Interface):
```

Comment:
Das ist doch eigentlich nur nen Implementierungsdetail der Formulare, das muss eigentlich nicht hier in die interfaces, oder?